db/blip_handler.go

```diff
@@ -552,9 +558,15 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 	return (err == nil && !forceClose), err
 }
 
-func (bh *blipHandler) buildChangesRow(change *ChangeEntry, revID string) []interface{} {
+func (bh *blipHandler) buildChangesRow(change *ChangeEntry, changeVersion ChangeByVersionType) []interface{} {
 	var changeRow []interface{}
 
+	// change map should only have one entry
+	revs := slices.Collect(maps.Values(changeVersion))
+	if len(revs) > 1 {
```

Comment:
`ChangeByVersionType` is just a map so you can take the length directly:

```suggestion
	if len(changeVersion) > 1 {
```